components/org.wso2.carbon.identity.oauth.ui/src/main/java/org/wso2/carbon/identity/oauth/ui/util/OAuthUIUtil.java

```diff
@@ -115,4 +115,13 @@ public static boolean isAudienceNotEmpty(String[] audiences) {
         }
         return false;
     }
+
+    /**
+     *  Replace space in the character name to set in id.
+     * @param name name to replaced
+     * @return replaced name
+     */
+    public static String replaceSpace(String name) {
```

Comment:
How about improving this method to return 
OAuthUIConstants.SCOPE_VALIDATOR_PREFIX + allowedValidator.replaceAll(" ", "_"));

And change the method to something like getScopeValidateDisplayName()

Reply:
yes. It's not used to display name, I will be getScopeValidateId() as it is used to set id. 

then I can create a new method getScopeValidateDisplayName() to set displace name